### ISSUE TITLE
rio up fails with invalid: metadata.ownerReferences if there are more than 8 services

### ISSUE THREAD

**Issue by lucidprogrammer** [rancher/rio#726]
**Describe the bug**
`
rio -n testx up --name some --file Riofile --answers Riofile-answers
`

`
FATA[0035] failed to create testx/web4 rio.cattle.io/v1, Kind=Service for  testx/some: Service.rio.cattle.io "web4" is invalid: metadata.ownerReferences: Invalid value: []v1.OwnerReference{v1.OwnerReference{APIVersion:"rio.cattle.io/v1", Kind:"Stack", Name:"some", UID:"660e9eaf-02a7-11ea-bdf1-021078f9eec5", Controller:(*bool)(0xc0137041b0), BlockOwnerDeletion:(*bool)(0xc0137041b1)}, v1.OwnerReference{APIVersion:"rio.cattle.io/v1", Kind:"Stack", Name:"some", UID:"660e9eaf-02a7-11ea-bdf1-021078f9eec5", Controller:(*bool)(0xc0137041b2), BlockOwnerDeletion:(*bool)(0xc0137041b3)}}: Only one reference can have Controller set to true. Found "true" in references for some and some, failed to create testx/web8 rio.cattle.io/v1, Kind=Service for  testx/some: Service.rio.cattle.io "web8" is invalid: metadata.ownerReferences: Invalid value: []v1.OwnerReference{v1.OwnerReference{APIVersion:"rio.cattle.io/v1", Kind:"Stack", Name:"some", UID:"660e9eaf-02a7-11ea-bdf1-021078f9eec5", Controller:(*bool)(0xc00df387e0), BlockOwnerDeletion:(*bool)(0xc00df387e1)}, v1.OwnerReference{APIVersion:"rio.cattle.io/v1", Kind:"Stack", Name:"some", UID:"660e9eaf-02a7-11ea-bdf1-021078f9eec5", Controller:(*bool)(0xc00df387e2), BlockOwnerDeletion:(*bool)(0xc00df387e3)}}: Only one reference can have Controller set to true. Found "true" in references for some and some 
`

**To Reproduce**
1. Try the attached Riofile, you should be able to reproduce the issue (it is renamed as Riofile.txt to allow uploading here to git issue)
2. Now try it again after commenting out from web9 onwards, it will work. So the only workaround is to create multiple Riofiles with less than 8 services, not ideal if you want to watch Riofile
[Riofile.txt](https://github.com/rancher/rio/files/3826937/Riofile.txt)

**Expected behavior**

**Kubernetes version & type (GKE, on-prem)**: `kubectl version`
```
Client Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.7-eks-1861c5", GitCommit:"1861c597586f84f1498a9f2151c78d8a6bf47814", GitTreeState:"clean", BuildDate:"2019-09-24T22:12:08Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.6-eks-5047ed", GitCommit:"5047edce664593832e9b889e447ac75ab104f527", GitTreeState:"clean", BuildDate:"2019-08-21T22:32:40Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}

```
Type:
**Rio version**: `rio info`
```
Rio Version: v0.6.0-alpha.4 (969baf29)
Rio CLI Version: v0.6.0-alpha.4 (969baf29)
Cluster Domain: 3fbqj3.on-rio.io
Cluster Domain IPs: 
System Namespace: rio-system
Wildcard certificates: true

```
**Additional context**
`rio systemlogs`
output:
```
rio-controller | time="2019-11-09T04:09:51Z" level=info msg="Starting rio-controller, version: v0.6.0-alpha.4, git commit: 969baf29"
rio-controller | time="2019-11-09T04:09:51Z" level=info msg="Updating CRD services.rio.cattle.io"
rio-controller | time="2019-11-09T04:09:51Z" level=info msg="Updating CRD stacks.rio.cattle.io"
rio-controller | time="2019-11-09T04:09:51Z" level=info msg="listening at :443"
rio-controller | I1109 04:09:51.960145       1 leaderelection.go:241] attempting to acquire leader lease  rio-system/rio...
rio-controller | I1109 04:09:51.967315       1 leaderelection.go:251] successfully acquired lease rio-system/rio
rio-controller | time="2019-11-09T04:09:53Z" level=info msg="Starting /v1, Kind=ConfigMap controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting apps/v1, Kind=Deployment controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting /v1, Kind=Secret controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting gitwatcher.cattle.io/v1, Kind=GitCommit controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting rio.cattle.io/v1, Kind=Stack controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting admin.rio.cattle.io/v1, Kind=ClusterDomain controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting apps/v1, Kind=StatefulSet controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting /v1, Kind=Service controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting rio.cattle.io/v1, Kind=ExternalService controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting rbac.authorization.k8s.io/v1, Kind=ClusterRoleBinding controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting apps/v1, Kind=DaemonSet controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting rbac.authorization.k8s.io/v1, Kind=ClusterRole controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting rio.cattle.io/v1, Kind=Service controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting tekton.dev/v1alpha1, Kind=TaskRun controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting gloo.solo.io/v1, Kind=Settings controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting cert-manager.io/v1alpha2, Kind=Certificate controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting extensions/v1beta1, Kind=Ingress controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting /v1, Kind=Endpoints controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting admin.rio.cattle.io/v1, Kind=PublicDomain controller"
rio-controller | time="2019-11-09T04:09:56Z" level=info msg="Starting rio.cattle.io/v1, Kind=Router controller"
```

**Comment by izaac** [comment]
Reproduced using `rio v0.6.0-alpha.4` in a k3s v0.10.2 cluster.  @StrongMonkey @daxmc99

**Comment by izaac** [comment]
thanks for reporting @lucidprogrammer

**Comment by StrongMonkey** [comment]
available to test in v0.6.0-alpha5.

**Comment by lucidprogrammer** [comment]
tested in v0.6.0-alpha5. it is fixed
